src/component/ResponsiveContainer.tsx

```diff
@@ -23,6 +23,8 @@ export interface Props {
   height?: string | number;
   minWidth?: string | number;
   minHeight?: string | number;
+  initalWidth?: number;
+  initalHeight?: number;
```

Comment:
could we pass these as an object where the object is optional but both properties `height` and `width` are required if specified? This way we can never have the height/width problem. You could make this `initialDimentions`?

Not terribly important but it does allow the API to say "if you want to set one of these you must set the other"

Reply:
I agree its a better expression of the intended behavior. I changed it.